spec/validators/edu_program_validator_spec.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+require 'rspec'
+
+describe EduProgramValidator do
+  describe 'when validating for load_csv context' do
```

Comment:
This validator doesn't know anything about `load_csv`.
Let's reword this something like 
```
context 'when record is unique and facility_code is present in Weam table' do
```

Reply:
changed